test/api.test.ts

```diff
@@ -0,0 +1,38 @@
+import fastify from 'fastify';
+import assert from 'assert';
+
+const app = fastify();
```

Comment:
```suggestion
import server from '../server'
```

Import server instead of create new app.